src/managers/RoleManager.js

```diff
@@ -132,6 +132,24 @@ class RoleManager extends BaseManager {
     return this.cache.get(this.guild.id);
   }
 
+  /**
+   * The premium subscriber role of the guild, if any
+   * @type {?Role}
+   * @readonly
+   */
+  get premiumSubscriberRole() {
+    return this.cache.find(role => role.premiumSubscriberRole);
+  }
+
+  /**
+   * The role associated with the client user of the guild, if any
+   * @type {?Role}
+   * @readonly
+   */
+  get myRole() {
+    return this.cache.find(role => role.myRole);
```

Comment:
```suggestion
    return this.cache.find(role => role.isMyRole);
```
typo missed in first review